### DISCUSSION TITLE
UniFi Script Update error

### DISCUSSION THREAD

**Discussion by diabolo511** [tteck/Proxmox#1455] [Help Needed]
Hey guys,

just tried to update my UniFi LXC but I end up in an error.

![Screenshot 2023-06-02 143608](https://github.com/tteck/Proxmox/assets/93436829/10bda0a9-6a8c-493e-b44a-43d91d5a3aa1)

Any Ideas how to fix it on-the-fly?

Thanks!

**Comment by tteck** [comment]
To update, just run these commands in the LXC console:
```
apt update
apt upgrade
```
